psm_utils/io/idxml.py

```diff
@@ -401,6 +437,9 @@ def _create_new_ids(self, psm_dict: dict) -> None:
                     peptide_id = oms.PeptideIdentification()
                     peptide_id.setMetaValue("spectrum_reference", spectrum_id)
                     peptide_id.setMetaValue("id_merge_index", msrun_reference.index(str(run)))
+                    # TODO: Question: Is there a way to infer the score type from the PSM?
```

Comment:
note: Unfortunately not. We could store it as a key in `provenance_data`. But then this feature would depend on the reader or user that generated the PSMList.